src/test/java/core/basesyntax/service/ParserTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import org.junit.Test;
+
+public class ParserTest {
+    private final Parser parserImpl = new ParserImpl();
+    private final Fruit apple = new Fruit();
+
+    @Test
+    public void parseLine_Ok() {
```

Comment:
please look through all your test methods in all test classes - they should follow the same naming pattern